scripts/g_r.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+import pandas
+from collections import Counter
+from sqlalchemy import create_engine
+engine = create_engine('mysql://root@localhost:3306/pantherbot_test', echo=False)
+
+def g_r(response, args):
+    if args[0] == 'time':
+        if len(args)!=4:
+            return ["ERROR: Please use the following syntax `time <channel> 2/1/17 5/1/18`"]
+        return time(args[2::], args[1])
+
+    if args[0] == 'top_users':
+        pass
+
+    if args[0] == 'emoji':
+        pass
+
+    if args[0] == 'channel':
```

Comment:
So then should this be seen as the groundwork for the analytics feature as appose to the analytics feature itself?